contracts/eden/include/members.hpp

```diff
@@ -68,11 +72,15 @@ namespace eden
                            election_rank,
                            representative,
                            encryption_key);
-      EDEN_FORWARD_FUNCTIONS(value, primary_key)
+      EDEN_FORWARD_FUNCTIONS(value, primary_key, by_representative)
    };
    EOSIO_REFLECT(member, value)
 
-   using member_table_type = eosio::multi_index<"member"_n, member>;
+   using member_table_type = eosio::multi_index<
+       "member"_n,
+       member,
+       eosio::indexed_by<"byrep"_n,
```

Comment:
The table exists in production: https://www.bloks.io/account/genesis.eden?loadContract=true&tab=Tables&table=member&account=genesis.eden&scope=0&limit=100

Reply:
Right... For some reason, I was thinking that it was okay because the index uses fields from member_v1 which doesn't exist in production yet.

Reply:
It'd be a win if we add a function to multi_index that rescans a table (at a provided scope value) and populates a new index

Reply:
I'm working on that now.

Reply:
Looking at the code, it looks as though multi_index will mostly work as long as you don't try to use the new index.  The only case that fails is a modify that changes the key used for the new secondary index.